bindings/rust/wasmedge-sdk/src/config.rs

```diff
@@ -1,14 +1,27 @@
 use super::wasmedge;
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
 pub struct Config {
     pub(crate) inner: wasmedge::Config,
 }
-
 impl Config {
-    pub fn with_wasi() -> Self {
-        let config = Self::default();
-        let inner: wasmedge::Config = config.inner.enable_wasi();
-        Self { inner }
+    pub fn create() -> Option<Self> {
+        let inner = wasmedge::Config::create();
+        match inner {
+            Ok(inner) => Some(Self { inner }),
+            Err(_) => None,
+        }
+    }
+}
+impl Config {
+    pub fn with_wasi() -> Option<Self> {
```

Comment:
May we use Result for this return value? Thanks.

Reply:
Accepted. Thanks!